src/array/_deprecated/find-last/find-last.ts

```diff
@@ -35,6 +35,7 @@ import { reverse } from '../reverse/index.js'
  * @typeParam T - The element type.
  *
  * @group Array
+ * @deprecated Use `xs.findLast(by)` instead.
```

Comment:
this doesn't exist, does it? :grin:

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findLast